e2e_correctness/test_modules.py

```diff
@@ -143,11 +153,15 @@ def _graphs_equal(memgraph_graph: Graph, neo4j_graph: Graph) -> bool:
     return True
 
 
-def _run_test(test_dir: str, memgraph_db: Memgraph, neo4j_driver: neo4j.BoltDriver):
+def _run_test(
+    test_dir: str, memgraph_db: Memgraph, neo4j_driver: neo4j.BoltDriver
+):
```

Comment:
If you return nothing, you can put
`-> None`
```suggestion
def _run_test(
    test_dir: str, memgraph_db: Memgraph, neo4j_driver: neo4j.BoltDriver
) -> None:
```